adapter/pkg/xds/xds_server.go

```diff
@@ -266,7 +266,9 @@ func UpdateAPI(byteArr []byte, upstreamCerts []byte, apiType string) {
 		// due to any modifications to the code.
 		logger.LoggerXds.Info("API type is not cuurently supported by WSO2 micro-gateway")
 	}
-
+	//:TODO: since currently labels are not taking from x-wso2-label, I have made it to be taken from the method
+	// argument.
```

Comment:
Shall we remove the new label assignment from the above code segment as well.

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/1595/commits/66f646dd975eb98c2efc9f6a7ce2d2a43391c0d7